db/warehouse/migrate/20230407164611_add_external_referrals.rb

```diff
@@ -0,0 +1,52 @@
+class AddExternalReferrals < ActiveRecord::Migration[6.1]
+  def change
+    create_table :unit_types do |t|
+      t.timestamps
+      t.string :description
+      t.integer :bed_type # HUD bed type
+      t.integer :unit_size
+      # checking... this would be external_id_id. Is this right?
+      t.references :external_id
+    end
```

Comment:
@gigxz are these specific to referrals or will we use the same unit types for "standard" units?

Reply:
They are not specific to referrals, they would be used for "standard" units too.

At the last meeting I suggested that we use the `external_ids` table for housing external IDs for unit types, too, but you objected to that.. I don't remember why. Do you? Or maybe I misunderstood you.

Reply:
Hmm, I don't remember why I objected.  If there's a `unit_types` table and we're getting the unit types from an external source or matching against one, then I think it makes sense.  I think I was thinking these weren't going to saved to a table, just in code, but I like this better, as long as the external reference is optional.

Reply:
Oh wait, looking back at the document, it seems like we _did_ decide that. The unit type table says `has_one :external_id` - that was probably confusing to write, sorry, but the intention was that the external_ids table has records where `source_type: Hmis::UnitType`. That will be a "belongs to" relationship for the unit type. I think you can go ahead and remove this `external_id ` column from the unit types table.

Reply:
I didn't see your message when I was writing my second one, but yeah. It is optional. @ttoomey you can remove this column.